runtime/vm/visible.c

```diff
@@ -271,16 +273,17 @@ setNestmatesError(J9VMThread *vmThread, J9Class *nestMember, J9Class *nestHost,
 /*
  * Loads and verifies the nesthost
  *
- * @param vmThread vmthread token
- * @param clazz the j9lass requesting the nesthost
- * @param options loading options (e.g. J9_LOOK_NO_THROW)
+ * @param[in] vmThread vmthread token
+ * @param[in] clazz the j9lass requesting the nesthost
+ * @param[in] options loading options (e.g. J9_LOOK_NO_THROW)
+ * @param[out] nestHostFound the nest host of clazz
  * @return 	J9_VISIBILITY_ALLOWED if nest host is loaded, otherwise
  * 			J9_VISIBILITY_NEST_HOST_LOADING_FAILURE_ERROR
  * 			J9_VISIBILITY_NEST_HOST_DIFFERENT_PACKAGE_ERROR
  * 			J9_VISIBILITY_NEST_MEMBER_NOT_CLAIMED_ERROR
  */
 UDATA
-loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
+loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options, J9Class **nestHostFound)
 {
 	UDATA result = J9_VISIBILITY_ALLOWED;
 	if (NULL == clazz->nestHost) {
```

Comment:
Suggest inverting this so the beginning of both branches are near the condition:
```c
	J9Class *nestHost = clazz->nestHost;
	if (NULL != nestHost) {
		*nestHostFound = nestHost;
	} else {
		J9ROMClass *romClass = clazz->romClass;
```
On the other hand, it makes it clearer that no visibility test occurs for this case: Does that make sense?